cmd/pop/main.go

```diff
@@ -480,6 +480,14 @@ func runFixupQueue(args fixupQueueArgs) error {
 		succeeded = append(succeeded, resp)
 	}
 
+	if errs != nil {
```

Comment:
Maybe use a `[]error` instead of `errors.Join` since this is how it is going to be consumed.

Reply:
done